src/dtos/group.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsString } from 'class-validator';
+
+export class CreateGroupDto {
+  @IsString()
+  public name: string;
+}
```

Comment:
1. Add Comments: You should consider adding comments to your class and its members. This would make your code more readable and easier to understand. For example, you could describe what this DTO is used for, what kind of group it creates, and also what name represents.

2. Add Additional Validators: Currently, you only ensure the name is a string. Depending on your application's requirements, you may need more detailed validation. For example, you might want to ensure the name is not empty and has a certain length. Class-validator provides many decorators for this:
```
import { IsString, Length, IsNotEmpty } from 'class-validator';

export class CreateGroupDto {
  @IsString()
  @Length(1, 50)
  @IsNotEmpty()
  public name: string;
}
```

3. Empty Line Between Decorator and Property Type Declaration: For readability, it would be better to have an empty line between the decorator and the type declaration of the property.